src/components/NavigationBar/index.js

```diff
@@ -326,6 +331,11 @@ class NavigationBar extends Component {
         checked={values && values.indexOf(name) > -1}
         value={name}
       >
+        {this.props.languageValue.indexOf(name) > -1 ? (
+          <Checkbox checked={true} style={checkIcon} />
```

Comment:
Why not this ?
```
  <Checkbox checked={this.props.languageValue.indexOf(name) > -1} style={checkIcon} />
```
It looks more compact

Reply:
Yeah Sure. That would make it compact. Thank you. I did the changes.